scripts/sync_s3_lifecycle.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+""" sync s3 lifecycle policy """
+
+import os
+
+import boto3
+
+
+def sync_s3_lifecycle():
+    s3 = boto3.resource('s3')
+    bucket_lifecycle_configuration = \
+        s3.BucketLifecycleConfiguration(os.environ.get('S3_BUCKET_NAME'))
+    bucket_lifecycle_configuration.put(
+        LifecycleConfiguration={
+            'Rules': [
+                {
+                    'Expiration': {
+                        'Days': 1465
```

Comment:
Does `1465` have a special meaning? Should it be configurable?

Reply:
My apologized, I went over the story requirements and realized there was no AC for object expiration. This has been removed.